### ISSUE TITLE
Pressing Enter doesn't work on Delete disks dialog

### ISSUE THREAD

**Issue by v-xuanzh** [microsoft/AzureStorageExplorer#1753]
**Storage Explorer Version:** 1.10.0
**Platform/OS:** Windows 10
**Architecture:** x64
**Commit:** f06deb7a6
**Regression From:** Not a regression

**Steps to reproduce:**
1. Expand 'Disks' node -> Open one resource group.
2. Create a new disk from a local VHD file.
3. Select the created disk then click 'Delete'.
4. On Delete dialog, press Enter directly -> Check the result.

**Expect Experience:**

The Enter key works well and the selected disk can be deleted successfully.

**Actual Experience:**

Nothing happens after pressing Enter on Delete dialog.

**Comment by MRayermannMSFT** [comment]
Fix merged into master
